vagrant/provisioning/roles/samba/tasks/main.yml

```diff
@@ -77,20 +88,20 @@
       - gcc 
       - gdb 
       - gnutls-devel 
-      - gpgme-devel 
+      - gpgme-devel #failed
```

Comment:
These packages shouldn't fail any more, after enabling powertools, right?

Reply:
I removed this comment in the upcoming commit, i forgot them in this one. And yes, they don't fail anymore.